### ISSUE TITLE
Question about producing the logits from Jetstream (context and generated output) like gather_all_token_logits in TRTLLM

### ISSUE THREAD

**Issue by salaki** [AI-Hypercomputer/JetStream#153]
Any example to set it up? Thanks
